core/src/commonMain/kotlin/com/revenuecat/purchases/kmp/PurchasesConfiguration.kt

```diff
@@ -25,13 +25,39 @@ public class PurchasesConfiguration private constructor(
                 "appUserId=$appUserId, " +
                 "purchasesAreCompletedBy=$purchasesAreCompletedBy, " +
                 "userDefaultsSuiteName=$userDefaultsSuiteName, " +
+                "storeKitVersion=$storeKitVersion, " +
                 "showInAppMessagesAutomatically=$showInAppMessagesAutomatically, " +
                 "store=$store, " +
                 "diagnosticsEnabled=$diagnosticsEnabled, " +
                 "dangerousSettings=$dangerousSettings, " +
                 "verificationMode=$verificationMode" +
                 ")"
 
+    internal fun storeKitVersionToUse(): StoreKitVersion {
+        var storeKitVersionToUse = this.storeKitVersion
+
+        if (this.purchasesAreCompletedBy is PurchasesAreCompletedBy.MyApp) {
+            storeKitVersionToUse = this.purchasesAreCompletedBy.storeKitVersion
+
+            if (this.storeKitVersion != StoreKitVersion.DEFAULT &&
+                storeKitVersionToUse != this.storeKitVersion) {
+                Purchases.logHandler.w("[Purchases]", "The storeKitVersion in purchasesAreCompletedBy " +
```

Comment:
Been thinking about logs in the hybrids for a while. We don't really have a great way of exposing these... So I think the solution in this PR works for now. But maybe we should create a top-level accessor for the `Purchases.logHandler`? Something like `fun handleLog(level, message)` that internally calls `Purchases.logHandler`...

In any case, I think this comment can be addressed separately cc @JayShortway

Reply:
Just curious, but what would be the advantage of a `handleLog()` function?

Reply:
Mostly having an indirection level lower than `Purchases.logHandler`... honestly it's nitpicking though, so I think it's also ok to leave as is!